src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1081,7 +1163,8 @@ core::Error runGrepOperation(const GrepOptions& grepOptions, const ReplaceOption
                              grepOptions.searchPattern,
                              grepOptions.directory,
                              grepOptions.asRegex,
-                             grepOptions.ignoreCase);
+                             grepOptions.ignoreCase,
+                             gitFlag);
```

Comment:
Which other filePattern information?

Reply:
excludeFilePatterns stores other data that gets added to the --exclude argument in the grep command, which is currently handled in processExcludeFilePatterns. We could pretty easily just call this function before initializing GrepOptions since it needs to be called at some point, but it seems a little weird because it is building actual arguments for the command, not just setting up general arguments.

Reply:
Looking at processExcludeFilePatterns, I do think those would make sense to be part of GrepOptions since all of those fields both change the grep options that will be used and impact nothing else, unless I'm misunderstanding and there isn't a unique GrepOptions per search (and optionally replace) request.

Reply:
Done 🎉